src/particles.js

```diff
@@ -46,58 +43,83 @@ class Particles extends Layer {
       },
       options
     );
+    this.pixelToGridRatio = 20;
+    this.tileSize = 1024;
+
     this.dropRate = 0.003; // how often the particles move to a random place
     this.dropRateBump = 0.01; // drop rate increase relative to individual particle speed
     this._numParticles = 65536;
+    // This layer manages 2 kinds of tiles: data tiles (the same as other layers) and particle state tiles
+    this._particleTiles = {};
+  }
+
+  visibleParticleTiles() {
+    return this.computeVisibleTiles(2, this.tileSize, {
+      minzoom: 0,
+      maxzoom: this.windData.maxzoom + 3 // how much overzoom to allow?
+    });
```

Comment:
Another function that just wraps a call to a one other function. Is it really necessary? I can understand the indirection if the wrapping function makes the code more readable.

Reply:
Well in this case it's nice because the parameters to the function are somewhat arbitrary, but they need to be the same in all call sites.